app/src/main/java/com/google/droidjump/Level.java

```diff
@@ -23,6 +23,7 @@
  * Represents a game level.
  */
 public class Level {
+
```

Comment:
revert

Reply:
But why? We have this blank in each file after class header so it's better to follow an existing style of the code.

Reply:
Usually we don't use a new line after each class header.

Reply:
Agreed, we usually don't do that. 

It was there before probably because it was overlooked. Now that we see it, let's fix it here and in other classes too.

Reply:
Fixed those in all files.